### ISSUE TITLE
The Leap-years has not been in mind

### ISSUE THREAD

**Issue by ehooo** [seatgeek/businesstime#12]
Hi, i think could be a problem in the Lead-years

https://github.com/seatgeek/businesstime/blob/master/businesstime/holidays/__init__.py#L9

**Comment by erwaller** [comment]
Yeah, that's definitely a known issue. Fortunately, it only affects holidays defined using the [`-week` specification](https://github.com/seatgeek/businesstime/blob/fc654cc89a78a51f2b5789666da69ac4251f9ce8/businesstime/holidays/__init__.py#L19) (which is used for holidays like Memorial Day which are defined by their relation to the end of the month) in February. I'm not aware of any holiday that matches that description yet so I've been thinking of this as relatively low priority.
